service/tablet_allocator.cc

```diff
@@ -1084,9 +1077,29 @@ class load_balancer {
                 .avg_tablet_size = avg_tablet_size,
                 .resize_decision = tmap.resize_decision(),
                 .tablet_count = tmap.tablet_count(),
-                .shard_count = shard_count
+                .shard_count = shard_count,
+                .min_tablet_count = 1,
             };
 
+            // FIXME: the table or the replication_strategy might be missing in boost unit tests
```

Comment:
What does this mean?

Reply:
That some boost unit tests forge a synthetic tablet map without populating the database with a respective keyspace, replication strategy, and table(s).

Reply:
@tgrabiec, is that fixable in your followup branch?

Reply:
> @tgrabiec, is that fixable in your followup branch?

It is, though I planned to fix existing tests in a follow-up.